core/src/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.11)
+cmake_minimum_required(VERSION 3.12)
```

Comment:
Nice addition.

Reply:
Need this for proper object library support (include object library to another target with target_link_library)